internal/update/update.go

```diff
@@ -32,8 +32,15 @@ type Release struct {
 	Timestamp   time.Time `yaml:"timestamp"`
 }
 
+// For disabling auto updates at build time
+var disableAutoUpdate = "false"
+
 // Check reports whether update checks should take place.
 func Check() bool {
+	if disableAutoUpdate == "true" {
+		return false
+	}
```

Comment:
this test can be blended into the `switch` statement below and would be more idiomatic

Reply:
👍🏾